### ISSUE TITLE
Confusion about enhancement function

### ISSUE THREAD

**Issue by xuhaomei** [byronBBL/DeCK#2]
Thank you for your contributions! 
In the paper, the enhancement function is defined as the sum of the frequency-weighted similarity and the logits (Figure 1). However, in the code (Figure 2), this does not appear to be the case. Could you please clarify this point?

<img width="1017" height="398" alt="Image" src="https://github.com/user-attachments/assets/392500b5-f9cb-424f-bf65-12e0362b8efd" />

<img width="1191" height="685" alt="Image" src="https://github.com/user-attachments/assets/9f84d569-16dc-4076-89a5-8420f6af2c60" />
